indico/modules/designer/placeholders.py

```diff
@@ -297,3 +299,36 @@ class RegistrationTicketQRPlaceholder(Placeholder):
     @classmethod
     def render(cls, registration):
         return generate_ticket_qr_code(registration)
+
+class RegistrationFieldPlaceholder(ParametrizedPlaceholder):
+    group = 'registration'
+    name = 'genericfield'
+    description = 'Generic registration field'
+    param_required = True
+    param_restricted = True
+    advanced = True
+
+    @classmethod
+    def render(cls, param, registration):
+        if ':' in param:
+            field_id, key = param.split(':', 1)
+        else:
+            field_id = param
+            key = None
+        import pdb; pdb.set_trace()
+        try:
+            fid = int(field_id)
+        except (TypeError, ValueError):
+            for avail_field in registration.registration_form.active_fields:
+                if avail_field.title == field_id:
```

Comment:
I don't think we should match by field title at all. We don't do so when emailing registrants either.

Reply:
Well, that was done on purpose here for two reason:
1) currently the user needs to enter this by hand, without good debuging knowledge the numeric value is hard to get.
2) When writing generic templates for more than a single registration form one can not match on the id here.

Getting available fields is planned as a follow-up

Reply:
1. Field titles are not necessarily unique, especially between sections - but 
2. Good point... that's actually a bigger problem I hadn't thought about. Can you join our IRC channel (#indico on freenode)? I think this (combined with (1)) needs more discussion and IRC is a better place for that